README.rst

```diff
@@ -16,12 +16,11 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
 The MDAnalysis project uses an `open governance model`_ and is fiscally sponsored by `NumFOCUS`_. Consider making 
 a `tax-deductible donation`_ to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.
 
-.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
-  :height: 60px
-  :target: https://numfocus.org/project/mdanalysis
-  :align: center
-  :alt: NumFOCUS
-  
+.. raw:: html
+
+   <img src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png" width="25%" alt="NumFOCUS">
```

Comment:
I see in the commit history that you tried several things, any reason (beyond flair) why centering was ditched here?

The template specifically refers to a centered logo, I don't know how much they care here though.

Reply:
None worked... =(

Reply:
Ok, the official template works as expected.